lms/djangoapps/teams/docs/decisions/0001-field-to-store-enabled-disabled-status.rst

```diff
@@ -0,0 +1,49 @@
+Add a new field to store enable/disabled status of teams
+--------------------------------------------------------
+
+Status
+======
+
+Proposal
+
+
+Context
+=======
+
+While implementing a UI to enable/disable teams we ran into issues due to the
+way teams is considered enabled or disabled, which is with the presence of one
+or more teamsets. So enabling requires adding a teamset and disabling requires
+removing all teamsets.
+
+Since we'd like to have a simple toggle to enable/disable teams this complicates
+the implementation significantly.
+
+Decisions
+=========
+
+We can add a new field called ``teams_enabled`` (or simply ``enabled``) to the
+``TeamsConfigField`` that stores the teams configuration in the course.
+
+Currently the ``is_enabled`` method on
+``openedx.core.lib.teams_config.TeamsConfig`` simply checks that the team_sets
+list isn't empty, it should switch to checking for the the ``enabled`` field
+instead. It can fall back to the existing behaviour so existing courses work
+as-is. However, any update to the this value will add this boolean field.
+
+Alternative
```

Comment:
I think this alternative is a lot more complicated. Adding a boolean field seems to be the way to go here